accepted-licenses.txt

```diff
@@ -0,0 +1,8 @@
+MIT
```

Comment:
I think this file should just be deleted... right? It is duplicated. The only one that should live on is the one inside the ``check-licenses`` folder

Reply:
Yes, that's right.